test/integration/targets/azure_rm_sql_server/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+- name: Create Servers
```

Comment:
I know you were saying you were having issues trying to get this to run but we should also look at expanding these tests a lot more, usually we look at performing 3 tasks per action, e.g. 3 tasks for creating a server, 3 tasks for modifying a server and 3 tasks for removing a server. The 3 tasks per action should follow;

* Run the task in check mode and check that the server wasn't actually created
* Run the task normally and check that the server was created as expected
* Run the task again (idempotent) and check that a change wasn't reported